src/test/regress/citus_tests/common.py

```diff
@@ -25,6 +25,8 @@
 from psycopg import sql
 from utils import USER
 
+IS_CITUS_VERSION_11_SQL = "SELECT (split_part(extversion, '.', 1)::int >= 11) as is_11 FROM pg_extension WHERE extname = 'citus';"
```

Comment:
Hmm, I cannot follow why we need this logic.

When do we need to run this test for Citus 11 or below? Are we considering to backport?

Reply:
With upgrade tests, when a citus cluster with an earlier version is set up `citus_set_coordinator_host` function does not exist. So I had to check citus version before calling the udf.